css/app.css

```diff
@@ -1,3 +1,913 @@
+    margin-bottom: 20px;
+  }
+
+  .latest-funding-top-right li:first-child a {
+    padding-left: 0;
+  }
+
+  .latest-articles {
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .main-article-showcase-header {
+    height: 22.5rem;
+  }
+
+  .showcase-item-header {
+    display: none;
+  }
+
+  .showcase-item-main {
+    justify-content: flex-end;
+  }
+
+  .showcase-grid-container {
+    height: 60vh;
+  }
+
+  .container {
+    width: 95vw;
+  }
+}
+
+@media (max-width: 767px) {
+  nav {
+    position: fixed;
+    z-index: 2;
+    height: 50px;
+  }
+
+  .navbar-container {
+    width: 100vw;
+    display: flex;
+  }
+
+  .nav-top {
+    display: none;
+  }
+
+  .nav-main {
+    display: none;
+  }
+
+  .nav-logo {
+    padding: 0;
+    height: 50px;
+  }
+
+  .nav-main-hidden {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    background-color: #fff;
+    width: 100%;
+    height: 50px;
+  }
+
+  .nav-main-hidden-right {
+    display: flex;
+    padding-right: 20px;
+  }
+
+  .nav-main-hidden-right .nav-main-search-sandwitch {
+    color: #f42;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+    width: 50px;
+  }
+
+  .showcase-grid-container {
+    height: 95vh;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr 1fr 1fr;
+    line-height: 1;
+    padding: 0.625rem;
+    padding-top: 60px;
+    gap: 0.625rem;
+  }
+
+  .showcase-grid-container * {
+    color: #fff;
+  }
+
+  .showcase-item {
+    position: relative;
+    height: 100%;
+    padding: 1.25rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    letter-spacing: -0.5px;
+    line-height: 1;
+  }
+
+  .showcase-title,
+  .showcase-item-main .showcase-title {
+    font-size: 1.3rem;
+  }
+
+  .showcase-item-main {
+    grid-column-start: 1;
+    grid-column-end: 2;
+    grid-row-start: 1;
+    grid-row-end: 2;
+  }
+
+  main {
+    padding: 10px;
+  }
+
+  .latest-news-grid-container,
+  .latest-funding-grid-container,
+  .latest-articles,
+  .latest-deals-grid-container {
+    grid-template-columns: 1fr;
+  }
+
+  .latest-news-article,
+  .latest-deals-item {
+    display: flex;
+  }
+
+  .latest-news-article img,
+  .latest-deals-item img {
+    width: 75px;
+    height: 75px;
+    margin-right: 20px;
+  }
+
+  .latest-deals-item img {
+    /* margin-right: 10%; */
+  }
+
+  .latest-deals-item-details {
+    padding-left: 10px;
+  }
+
+  .latest-funding-top {
+    flex-direction: column;
+  }
+
+  .latest-funding-subtitle {
+    font-size: 1.2rem;
+  }
+
+  .latest-funding-top-right {
+    margin-bottom: 20px;
+  }
+
+  .latest-funding-title {
+    font-size: 1.4rem;
+  }
+
+  .latest-funding-footer,
+  .latest-funding-content {
+    font-size: 1rem;
+  }
+
+  .latest-articles .main-article .main-article-showcase-header {
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .latest-deals-discount-badge {
+    display: none;
+  }
```

Comment:
Same style definition of selectors on lines 771 and 775, group selectors with the same style definition, or create a new class.